pylsp/config/schema.json

```
@@ -87,6 +87,14 @@
       "uniqueItems": true,
       "description": "List of errors and warnings to enable."
     },
+    "pylsp.plugins.jedi.auto_import_modules": {
+      "type": "array",
+      "default": ["numpy", "gi"],
```

```suggestion
      "default": ["numpy"],
```

Why `gi` here? I think that's specific to Linux only, so we shouldn't include it by default here. Besides, you can easily add it from your client.